package.json

```diff
@@ -11,6 +11,7 @@
     "url": "git+https://github.com/probot/serverless-lambda.git"
   },
   "devDependencies": {
+    "@types/jest": "^26.0.0",
```

Comment:
Can you undo this change, and the change to the `package-lock.json`? We don't need `@types/jest`